cmd/rosa/main.go

```diff
@@ -94,3 +102,25 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func CheckROSACliVersion() error {
+	latestTag, err := ocm.GetLatestROSACliVersion()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error checking latest version: %s\n", err)
+	}
+	if latestTag != "" && info.Version != latestTag {
+		latestVersion, err := semver.NewVersion(latestTag)
+		if err != nil {
+			return err
+		}
+		currentVersion, err := semver.NewVersion(info.Version)
+		if err != nil {
+			return err
+		}
+		if currentVersion.LessThanOrEqual(latestVersion) {
+			return fmt.Errorf("A newer version of rosa cli '%s' is available for download. Goto "+
+				" 'https://github.com/openshift/rosa/tags' if you wish to update", latestTag)
```

Comment:
Is this the official way of updating the cli?

Reply:
No. The official URL to download is https://www.openshift.com/products/rosa/download

Reply:
Alternatively, we already have a `rosa download` command, I think part of the epic calls for a `rosa download rosa` command to download the rosa client itself.

Reply:
@vkareh @tbrisker  We can certainly add rosa update --version command or rosa download rosa but for now i have added the generic message and pointed to the official download page. can you please check